mlflow/server/js/src/utils/Utils.js

```diff
@@ -6,12 +6,12 @@ import laptopSvg from '../static/laptop.svg';
 import projectSvg from '../static/project.svg';
```

Comment:
It looks like your IDE reformatted this whole file unfortunately. Can you reduce the diff to just your changes without changing the other lines?

Reply:
Yes. I'll look into this and do the needful. Thanks